common/src/main/java/io/github/steveplays28/stevesrealisticsleep/mixin/ServerWorldMixin.java

```diff
@@ -109,9 +109,25 @@ protected ServerWorldMixin(MutableWorldProperties properties, RegistryKey<World>
 	private boolean stevesrealisticsleep$shouldSkipWeather = false;
 	@Unique
 	private int stevesrealisticsleep$consecutiveSleepTicks = 0;
+	@Unique
+	private int stevesrealisticsleep$ticksSinceLastChecked = 0;
```

Comment:
Could be more descriptive, e.x.:
```suggestion
	private int stevesrealisticsleep$ticksSinceLastTpsCheck = 0;
```